### DISCUSSION TITLE
How to enable user-data-dir in firefox?

### DISCUSSION THREAD

**Discussion by ghost** [wxt-dev/wxt#1059] [Q&A]
Hi there,

We’re really enjoying using WXT! When working in Firefox, we’d like to achieve functionality similar to Chrome’s user-data-dir, so our data remains saved even when the dev server is turned off. How can we do that?

```
chromiumArgs: ['--user-data-dir=./.dev-profile'],
```

**Comment by aklinker1** [comment]
`web-ext` I don't think Firefox has a CLI flag like that, so I don't think it's possible.
